dynamo-rest/src/main/java/org/dynamoframework/rest/crud/CrudController.java

```diff
@@ -183,7 +197,12 @@ public T put(@PathVariable("entityName") String entityName, @PathVariable("id")
                     .formatted(entityName, id));
         }
 
-        T source = objectMapper.readerFor(clazz).readValue(request);
+        T source = null;
+        try {
+            source = objectMapper.readerFor(clazz).readValue(request);
+        } catch (JsonProcessingException e) {
```

Comment:
Is het niet mooier om die JsonProcessingException direct op te vangen in de exception handler? Dan hoef je niet iedere keer een try-catch te gebruiken